src/sentry/static/sentry/app/views/settings/organization/permissionAlert.tsx

```diff
@@ -12,7 +13,7 @@ const PermissionAlert = ({access = ['org:write'], ...props}: Props) => (
   <Access access={access}>
     {({hasAccess}) =>
       !hasAccess && (
-        <Alert type="warning" icon="icon-warning-sm" {...props}>
+        <Alert type="warning" icon={<IconWarning size="sm" />} {...props}>
```

Comment:
Actually feel like I would just leave off the size. IMO this is too small now that i look closer

Reply:
lol why didn't you mention this earlier? But the thing Chis pointed me to doesn't have icons of size 24: https://www.figma.com/file/Gq0SG3GrwBu3S1NArPjB5bEo/Sentry-Design-System?node-id=3257%3A0

Reply:
lol true, okay okay